index.js

```diff
@@ -156,5 +164,7 @@ function gql(/* arguments */) {
 gql.default = gql;
 gql.resetCaches = resetCaches;
 gql.disableFragmentWarnings = disableFragmentWarnings;
+gql.print = require('graphql/language/printer');
+gql.parser = parser
```

Comment:
@stubailo I did a search:
https://github.com/apollographql/apollo-client/search?utf8=%E2%9C%93&q=parser

and saw some typings expecting `parser` 

What do you suggest? `parse`?